aries-site/src/pages/design-tokens/color-pairing.mdx

```diff
@@ -0,0 +1,66 @@
+import { Box, Grid, Notification, Text } from 'grommet';
+import { Trigger } from 'grommet-icons';
+import { ContentPane, Example } from '../../layouts';
+import { ColorPreview } from '../../components';
+import {
+  SpecialtyBackgrounds,
+  StandardBackgrounds,
+  StandardForegrounds,
+  StrongBackgrounds,
+} from '../../examples';
+
+## Using background and foreground colors
+
+Backgrounds pair with text and icon colors, unless they have a specific color pairing. For example, `color.background.default` or `color.background.front` will work with `color.text.default` or `color.icon.strong`.
+
+## Accessible color pairings
```

Comment:
Since all background and foreground colors are accessible color pairings and this is just speaking to a subset, it feels a little awkard.